src/markeritem.cpp

```diff
@@ -26,6 +26,11 @@ QColor MarkerItem::color() const
   return brush().color();
 }
 
+void MarkerItem::setHighlight(const QColor& color)
```

Comment:
I mentioned this earlier, but I'd prefer to see a semantic flag instead of a generic color property.

Reply:
I remembered you mentioned that but I wasn't sure the best way to go about it. This is more a placeholder solution.